rules/game_process.tex

```diff
@@ -340,6 +364,19 @@ \subsection{Free Kick}
 Note that in the case of the Pushing free kick the sub-state is activated automatically through the ``Foul''.
 The team who is awarded the free kick (termed the attacking team) has \qty{\FreeKickTime}{\second} to complete the kick.
 
+In Champions Cup, the referee communicates which team is to take the free kick via a visual gesture.
+The GameController packet will tell the robots which free kick sub-state is active, but not who is to kick the ball.
+\unsure{This gesture is entirely tentative and just my (Francesco's) proposal, pending discussion with the TC. The intent is to mirror the FIFA gestures in a simplified way.}
+The gesture consists of raising one arm horizontally and sideways, parallel to the field line, while the other arm remains at rest.
+The raised arm is the one pointing towards the defending team's goal, i.e. the goal of the team that is \textit{not} taking the free kick.
+\textit{(For reference, the gesture is the same as the "Kick-in" gesture from the 2023 Visual Referee Challenge, but the opposite arm is raised with respect to back then.)}
+This gesture is made by the head referee and held for at least 5 seconds after the GameController signal is sent.
+\textit{(The referee may look at the GameStateVisualizer monitor to check that the GC signal is sent and when the time allotted for the gesture has passed.)}
+The referee sends the signal standing as close as possible to the T-junction, but their actual positioning along the sideline may deviate from this standard position, even significantly,
```

Comment:
Discuss: might this mean that the head ref won't be able to stand behind the goal (physical structure) to adjudicate close goals (points), where the ball barely crosses the line? Can/Should an assistant do this?

Reply:
I would say there should be mentioned something that the head referee should try to reach the T-Junction after calling a set play. For a goal its not a problem since no set play follows but for a goalkick or corner kick this could happen that the ref deviated a bit from the T-Junction. 

We also had the idea that there might be a 5th referee involved.